lib/reporting/report_row_builder.rb

```diff
@@ -27,7 +27,7 @@ def build_row(item)
     end
 
     def slice_and_format_row(row)
-      result = row.to_h.reject { |k, _v| k.in?(report.fields_to_hide) }
+      result = row.to_h.select { |k, _v| k.in?(report.fields_to_show) }
```

Comment:
This is probably easier for the feature toggle but otherwise I would have thought that we can simplify it:
```rb
result = row.to_h.slice(report.fields_to_show)
```

Reply:
Yeap. I maybe find out more readable in the case of feature toggle (`select`/`reject`):

```ruby
if OpenFoodNetwork::FeatureToggle.enabled?(:report_inverse_columns_logic, @current_user)
 row.to_h.select { |k, _v| k.in?(report.fields_to_show) }
else
 row.to_h.reject { |k, _v| k.in?(report.fields_to_hide) }
end
```